website/developer-docs/blueprints/v1/tags.md

```diff
@@ -10,7 +10,11 @@ If no matching entry can be found, an error is raised and the blueprint is inval
 
 #### `!Find`
 
-Example: `configure_flow: !Find [authentik_flows.flow, [slug, default-password-change]]`
+Example:
+
+`configure_flow: !Find [authentik_flows.flow, [slug, default-password-change]]`
+
+`configure_flow: !Find [authentik_flows.flow, [!Context property_name, !Context property_value]]`
```

Comment:
I wonder if these more complicated tags can be broken up into multiple lines to make it a bit easier to read